src/csharp/Grpc.Microbenchmarks/CommonThreadedBase.cs

```diff
@@ -35,7 +35,7 @@ public abstract class CommonThreadedBase
     {
         protected virtual bool NeedsEnvironment => true;
 
-        [Params(1, 1, 2, 4, 8, 12)]
```

Comment:
FYI, this was copy-pasta from the old `RunCompletionRegistryBenchmark` / `RunPInvokeByteArrayBenchmark` / `RunSendMessageBenchmark` methods

Reply:
I am aware of that. I guess the `1, 1` was there as a makeshift "warmup" phase originally, so I removed it.